packages/cli/src/utils/helpers.ts

```diff
@@ -11,7 +10,20 @@ export function enumEntriesDupWithLowercase<T>(entries: [string, T][]) {
   return enumMap
 }
 
-export async function nodeIsSynced(web3: Web3): Promise<boolean> {
+export type EthSyncingRpcSchema = {
+  Parameters: []
+  Method: 'eth_syncing'
+  ReturnType: false | { startingBlock: bigint; currentBlock: bigint; highestBlock: bigint }
+}
+
+export async function nodeIsSyncedRaw(client: CeloClient) {
```

Comment:
its slightly bothering me that the rpc method is `is-syncing` but the function is `is-synced` 

when i read it it sounds like if it is in the process of syncing it is not yet synced. but if i think about it then it does make sense because it should always be syncing and syncing means up to date and synced with the chain.

Reply:
The only place it's called is `packages/cli/src/commands/node/synced.ts` and it used to be `web3.eth.isSyncing` so i'll renamed accordingly

Reply:
actually i think im correct. is-syncing means its not synced 

> Ethereum API method that returns an object with information about the current synchronization status of the node, or false if the node is fully synced. "

Reply:
done